common4j/src/main/com/microsoft/identity/common/java/exception/ClientException.java

```diff
@@ -393,6 +393,11 @@ public class ClientException extends BaseException {
      */
     public static final String UNSUPPORTED_ANDROID_API_VERSION = "unsupported_android_api_version";
 
+    /**
+     * The android version used does not support the operation.
```

Comment:
The javadoc doesn't seem to match with the actual String here. The actual string seems to indicate that something was null

Reply:
update common